Patches/Core/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -640,36 +640,37 @@
 		<statBases>
 			<Mass>8</Mass>
 			<RangedWeapon_Cooldown>0.43</RangedWeapon_Cooldown>
-			<SightsEfficiency>1</SightsEfficiency>
+			<SightsEfficiency>1.1</SightsEfficiency>
 			<ShotSpread>0.15</ShotSpread>
 			<SwayFactor>1.8</SwayFactor>
 			<Bulk>10</Bulk>
-			<WorkToMake>39500</WorkToMake>
+			<WorkToMake>35500</WorkToMake>
 		</statBases>
 		<costList>
 			<Steel>65</Steel>
 			<Plasteel>30</Plasteel>
-			<ComponentIndustrial>7</ComponentIndustrial>
+			<ComponentIndustrial>1</ComponentIndustrial>
+			<ComponentSpacer>1</ComponentSpacer>
```

Comment:
The incendiary launcher is Industrial tech level.

Reply:
CE's advanced launchers tech is Spacer, the T-9 uses spacer ammo and is even considered spacer in the spreadsheets.